security/token-file-provider.1.rst

```diff
@@ -0,0 +1,164 @@
+NAME
+====
+token-file-provider – Generate Vault tokens for EdgeX services
+
+
+SYNOPSIS
+========
+token-file-provider [-h|-help] [-confdir confdir]
+
+
+DESCRIPTION
+===========
+token-file-provider generates per-service Vault tokens for EdgeX services
+so that they can make authenticated connections to Vault to retrieve
+application secrets.
+token-file-provider implements a generic secret seeding mechanism based
+on pre-created files and is designed for maximum portability.
+token-file-provider takes a configuration file that specifies the services
+for which tokens shall be generated and the Vault access policy
+that shall be applied to those tokens.
+token-file-provider assumes that there is some underlying protection mechanism
+that will be used to prevent EdgeX services from reading each other’s tokens.
+
+
+OPTIONS
+=======
+  \-h
+    Display help text
+
+  \-help
+    Display help text
+
+  \-confdir confdir
```

Comment:
I think we should follow the rest of EdgeX convention for this and have the following set of options that together control the location of the configuration.toml:

- `-c` 
- `-confdir`
- `-p`
- `-profile`

Reply:
The usages in `edgex-go/internal/pkg/usage/usage.go` are a little inconsistent.  Asking for double-confirmation that this is the right list.

Reply:
Resolved in https://github.com/edgexfoundry/edgex-docs/pull/35/commits/bd99b6ff553c24070f5265d0c5914089d399ff3f